docs/source/fury-pybullet.rst

```diff
@@ -0,0 +1,606 @@
+
+*Note: keeping track of physics client ID is optional unless multiple physics clients are used. In order to observe the same simulation in pybullet, replace p.DIRECT with p.GUI.*
+
+.. _disconnection:
+
+Disconnection
+-------------
+
+PyBullet Physics client can be shutdown by the following command:
+
+.. code-block:: python
+
+  p.disconnect()
+
+.. _gravity:
+
+Setting Gravity
+---------------
+
+Global Scene gravity can be set using the following command:
+
+.. code-block:: python
+
+  # Gravity vector.
+  gravity_x = 0
+  gravity_y = 0
+  gravity_z = -10
+  p.setGravity(gravity_x, gravity_y, gravity_z)
+
+.. _objects:
+
+Creating Objects
+----------------
+
+The following criterion must be fulfilled in order to create an object which is in sync with both Fury and pyBullet:
+
+
++-----------------------+----------------------------------------------------------------------+
+|       Object Actor    |         The actor which will be rendered by Fury                     |
++-----------------------+----------------------------------------------------------------------+
+|      Collision Shape  |  The shape used by pybullet for collision simulations.               |
+|                       |  **Optional** if collision simulation is not required.               |
++-----------------------+----------------------------------------------------------------------+
+|       Multi-Body      |  The object that will be tracked by pybullet for general simulations.|
++-----------------------+----------------------------------------------------------------------+
+
+The following is a snippet for creating a spherical ball of radius = 0.3
+
+.. code-block:: python
+
+  ###### Creating BALL
+  # Ball actor
+  ball_actor = actor.sphere(centers = np.array([[0, 0, 0]]),
+                            colors=np.array([1,0,0]),
+                            radii=0.3)
+
+  # Collision shape for the ball.
+  ball_coll = p.createCollisionShape(p.GEOM_SPHERE,
+                                     radius=0.3)
+
+  # Creating a Multibody which will be tracked by pybullet.
+  ball = p.createMultiBody(baseMass=3,
+                           baseCollisionShapeIndex=ball_coll,
+                           basePosition=[2, 0, 1.5],
+                           baseOrientation=[ 0, 0, 0, 1 ])
+
+*Note: Centers for the actor must be set to (0, 0, 0) or else the simulation will be offset by that particular value.*
+
+.. _change_dynamics:
+
+Changing Object Dynamics
+------------------------
+
+Object dynamics such as mass, lateral_friction, damping, inertial_pos, inertial_orn, restitution, rolling friction etc can be changed. The following snippet shows how to change the lateral_friction and coeff of restitution of the same ball:
+
+.. code-block:: python
+
+  p.changeDynamics(ball, -1, lateralFriction=0.3, restitution=0.5)
+
+*Note: The second parameter is linkIndex which is for bodies having multiple links or joints. Passing -1 means applying changes to the base object.*
```

Comment:
`..note:: *The second parameter is linkIndex which is for bodies having multiple links or joints. Passing -1 means applying changes to the base object.*`